Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1865,6 +1866,160 @@ public static void HandleMagicGod(Session session, params string[] parameters)
             ChatPacket.SendServerMessage(session, "You are now a magic god!!!", ChatMessageType.Broadcast);
         }
 
+        
+        [CommandHandler("modifyvital", AccessLevel.Admin, CommandHandlerFlag.None, 2, "Adjusts the maximum vital attribute for the last appraised mob/player and restores full vitals", "<Health|Stamina|Mana> <delta>")]
+        public static void HandleModifyVital(Session session, params string[] parameters)
+        {
+            var lastAppraised = CommandHandlerHelper.GetLastAppraisedObject(session);
```

Comment:
my only concern about GetLastAppraisedObject() is how decal can automatically appraise other objects invisibly / behind-the-scenes, and unintentionally change the intended target of the player

there is also a field in ACE which points to the 'currently selected creature target' (anything with vitals), and this selection is not automatically changed by decal, as far as i know... perhaps it would be worth considering to use that variant?

Reply:
This seems like a good idea. I found the `selectedTarget` field in Player class but it's a Guid. What's the easiest way to convert that to a Creature instance?

Reply:
Something like this should work:
Creature targetCreature = CurrentLandblock?.GetObject(Guid) as Creature;